### DISCUSSION TITLE
debug help messages discussion

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2162] [General]
![image](https://user-images.githubusercontent.com/54838975/200019243-14cfa2cb-2741-4ab8-9747-1dd7fb27410d.png)

well it's a [meme](https://www.bbc.com/news/technology-47900904) but its context gets lost in translation, therefore these two options should be viable:

1. make debug help message generic (no memes)
2. unique debug help messages per translation languages

**Comment by silverwing235** [comment]
Option#1 for me, I think.

**Comment by chaosvolt** [comment]
It's a fairly niche meme in my opinion and definitely seems evocative of DDA's attitude towards gameplay anyway, so a more conventional warning message is probably better. Unless we can replace it with a meme that translates well, in which case I'd say keeping it a meme is fine but maybe one with less of a "boo hoo you're cheating when there's a 90% chance you're doing this either to test something or because the game did something weird you now need to fix" feel to it.
